src/api.js

```diff
@@ -0,0 +1,61 @@
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+async function createGame() {
+  // Check if you have already created a game before
+  if (localStorage.getItem('gameId')) return;
+
+  // If you haven't, create a new game
+  const game = {
+    name: 'Awesome Game',
+  };
+
+  const response = await fetch(`${baseUrl}games/`,
+    {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(game),
+    });
+  const data = await response.json();
+  const gameId = data.result.split(' ')[3];
+  localStorage.setItem('gameId', gameId);
+}
+
+async function refresh() {
+  const gameId = localStorage.getItem('gameId');
```

Comment:
- [ ] Kindly try to use the arrow functions instead of the function keyword. Arrow functions provide a simpler and more concise way to define functions. Kindly change the functions to arrow functions.